packages/redux-send-postMessage-thunk/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "reduxSendPostMessageMiddleware",
+  "version": "1.0.0",
+  "description": "A redux middleware that allows consumer to conditionally dispatch messages to window.postMessage",
+  "main": "dist/sendPostMessageMiddleware.cjs.js",
+  "module": "dist/sendPostMessageMiddleware.esm.js",
+  "files": [
+    "dist",
+    "index.d.ts"
+  ],
+  "types": "index.d.ts",
+  "scripts": {
+    "test": "NODE_ENV=test node node_modules/.bin/jest",
+    "build": "rollup -c && tsc --emitDeclarationOnly",
+    "pretest": "yarn run build"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
All the rest of the modules in this repo are MIT licensed, and this will be too.  If you want to put yourself as the author, that's fine with me.  If not, you can add `ian.b.taylor@gmail.com` as the author, like the rest of the packages in the repo.

Reply:
Fixed this. Also updated the dependencies, keywords and scripts to be more in line with the other packages.